boudicca.base/publisher-event-html/src/main/kotlin/base/boudicca/publisher/event/html/controller/StartPageController.kt

```diff
@@ -14,7 +14,7 @@ import org.springframework.web.servlet.ModelAndView
 @RequestMapping("/")
 class StartPageController @Autowired constructor(private val eventService: EventService) {
 
-    private val PAGE_TITLE = "Boudicca"
+    private val PAGE_TITLE = "Boudicca.Events - Open source event collector and publisher software"
```

Comment:
hmm
do we want to focus on events or on our software here? i think the former, so i would focus on that

something mentioning more our loval events, or accessibility or something like that

Reply:
Boudicca.event - find accessible events in austria ?

Reply:
yeah something like that! :)